mods/cnc/chrome/lobby-kickdialogs.yaml

```diff
@@ -10,23 +10,18 @@ Background@KICK_CLIENT_DIALOG:
 			Height: 25
 			Font: Bold
 			Align: Center
-		Label@TEXTA:
+		Label@TEXT:
```

Comment:
These changes regress the layout ingame:
Before:
<img width="371" alt="before" src="https://github.com/user-attachments/assets/8fd274a2-9e77-40f1-a262-5604d64dfb73" />

After:
<img width="436" alt="after" src="https://github.com/user-attachments/assets/98549e56-0044-4fcf-a3d7-dd580450ef32" />

Reply:
reverted